CHANGELOG.rst

```diff
@@ -522,7 +522,7 @@ v.2.8.10 2022-01-19
 
 Fixes:
 
-* Add timeouts to requests calls (see :ref:`ckan.requests.timeout`) (`#6408 <https://github.com/ckan/ckan/pull/6408>`_)
+* Add timeouts to requests calls (see `ckan.requests.timeout`) (`#6408 <https://github.com/ckan/ckan/pull/6408>`_)
```

Comment:
Is removing this link supposed to be part of this PR?

Reply:
There is no corresponding reference in the documentation and that leads to a warning during documentation build, which prevents tests from passing. 

So this problem is not related to this particular PR but without this change none of the new PRs can pass the tests